### ISSUE TITLE
PHONEMes Question

### ISSUE THREAD

**Issue by ucsdwxu** [saezlab/PHONEMeS#1]
Hi Enio,

I had a look at the PHONEMES page and may I ask whether it is compulsory for it to work on cluster? Also in the scripts for those current examples you have on the GitHub, to what extent I need to modify the content within the script for my own use?

Thank you.
Cheers,
Weijun

**Comment by enio23** [comment]
Hello,

There is indeed a way to make it work on a standalone machine through an ILP formulation. For that however you might have to download CPLEX solver from IBM (https://ibm.onthehub.com/WebStore/OfferingDetails.aspx?o=6fcc1096-7169-e611-9420-b8ca3a5db7a1) which is free for academic purposes. The ILP implementation is not public yet, however if you have a case-study where you would like to use PHONEMeS, I would be happy to help you on that direction :)

Cheers,
Enio

**Comment by ucsdwxu** [comment]
Hi Enio,

That's very good to know you are happy to help me out for my direction. I will think about what exactly I will plan to do and contact you again if I need further help. Have a nice day.

Best regards,
Weijun

**Comment by ucsdwxu** [comment]
> Hello,
> 
> There is indeed a way to make it work on a standalone machine through an ILP formulation. For that however you might have to download CPLEX solver from IBM (https://ibm.onthehub.com/WebStore/OfferingDetails.aspx?o=6fcc1096-7169-e611-9420-b8ca3a5db7a1) which is free for academic purposes. The ILP implementation is not public yet, however if you have a case-study where you would like to use PHONEMeS, I would be happy to help you on that direction :)
> 
> Cheers,
> Enio

Hi Enio,

I tried to visit the weblink for downloading CPLEX but it seems that the link is not working. Also may I know whether for PhoneMeS calculation whether the cluster computing is a necessary requirement? If on a local machine, would you have any idea about the computing power needed? I am still planning on my case study now, but for me to start on it, do I need a Prior Knowledge Network model as a beginning point?

Thank you very much for your help again.
Best regards,
Weijun

**Comment by enio23** [comment]
Hello,

Sorry for the inconvenience, that link was indeed broken. Can you please try out this other one? https://www.ibm.com/developerworks/community/blogs/jfp/entry/CPLEX_Is_Free_For_Students?lang=en

Also for PHONEMeS you need a prior knowledge, however since PHONEMeS is a method built to handle large scale phospho-proteomics, you might need to use databases of interactions in order to have maximal coverage. We have also been working on that direction in our lab where we have developed Omnipath - a comprehensive collection of literature curated human signaling pathways and signalling interactions. For more about Omnipath and would suggest you to check out the following links:

http://omnipathdb.org/
https://www.nature.com/articles/nmeth.4077
http://omnipathdb.org/ptms/?fields=sources&fields=references&genesymbols=1
http://omnipathdb.org/interactions/?fields=sources&fields=references&genesymbols=1

Basically what PHONEMeS needs as a prior knowledge network to train is a list of K/P-S interactions (i.e. from link 4) which can further be complemented with a list of directed protein interactions (i.e. from link 5) in order to build the logic models of perturbation flow.

Hope the explanation helped you clarify a little bit more and for any other requests, please do not hesitate to contact me.

Cheers,
Enio

**Comment by ucsdwxu** [comment]
> Hello,
> 
> Sorry for the inconvenience, that link was indeed broken. Can you please try out this other one? https://www.ibm.com/developerworks/community/blogs/jfp/entry/CPLEX_Is_Free_For_Students?lang=en
> 
> Also for PHONEMeS you need a prior knowledge, however since PHONEMeS is a method built to handle large scale phospho-proteomics, you might need to use databases of interactions in order to have maximal coverage. We have also been working on that direction in our lab where we have developed Omnipath - a comprehensive collection of literature curated human signaling pathways and signalling interactions. For more about Omnipath and would suggest you to check out the following links:
> 
> http://omnipathdb.org/
> https://www.nature.com/articles/nmeth.4077
> http://omnipathdb.org/ptms/?fields=sources&fields=references&genesymbols=1
> http://omnipathdb.org/interactions/?fields=sources&fields=references&genesymbols=1
> 
> Basically what PHONEMeS needs as a prior knowledge network to train is a list of K/P-S interactions (i.e. from link 4) which can further be complemented with a list of directed protein interactions (i.e. from link 5) in order to build the logic models of perturbation flow.
> 
> Hope the explanation helped you clarify a little bit more and for any other requests, please do not hesitate to contact me.
> 
> Cheers,
> Enio

Dear Enio,

Thank you so much for the detailed explanation and information guide. In my case, I may not be looking for a logical model after enzyme inhibitor perturbation but instead I am interested in looking at how the signaling pathways and phosphoproteome is altered after the cells respond to an external stimulus (cue). Would you think PhoneMes would do a similar job as compared to perturbation case?

Thank you again.
Best,
Weijun

**Comment by enio23** [comment]
Hi,

Yes. We have case studies where we used PHONEMeS to build our models from an activated receptor by a stimulus/ligand to downstream signalling pathways it activates.

Cheers,
Enio
